lib/native/source/opensmt/swigWrapper.patch

```diff
@@ -0,0 +1,416 @@
+diff --git a/lib/native/source/opensmt/opensmt-wrap.cpp b/lib/native/source/opensmt/opensmt-wrap.cpp
```

Comment:
OpenSMT is licensed with MIT license.
Should we publish all the JavaSMT bindings for OpenSMT as `Apache2.0 OR MIT`, or only the C files?